docs/src/tutorials/cauer_low_pass_analog.md

```diff
@@ -0,0 +1,250 @@
+        resistor2 = Resistor(R=1)
+        inductor1 = Inductor(L=L1)
+        inductor2 = Inductor(L=L2)
+        capacitor1 = Capacitor(C=C1)
+        capacitor2 = Capacitor(C=C2)
+        capacitor3 = Capacitor(C=C3)
+        capacitor4 = Capacitor(C=C4)
+        capacitor5 = Capacitor(C=C5)
+    end
+    @equations begin #= ... =# end
+end
+```
+
+#### Defining Model Equations
+
+Defining the equations simply requires connecting all the components.
+This is done with the `connect` method and knowledge of the component ports.
+If you are unsure what ports are available, see the components help section named "Connectors" to learn more.
+
+```julia
+@mtkmodel CauerLowPassAnalog begin
+    @parameters begin #= ... =# end
+    @components begin #= ... =# end
+    @equations begin
+        connect(step.output, source.V)
+        connect(resistor1.n, capacitor1.p)
+        connect(capacitor1.n, ground.g)
+        connect(inductor1.p, capacitor2.p)
+        connect(inductor1.p, capacitor1.p)
+        connect(inductor1.n, capacitor2.n)
+        connect(capacitor2.n, capacitor3.p)
+        connect(capacitor2.n, capacitor4.p)
+        connect(capacitor2.n, inductor2.p)
+        connect(inductor2.n, capacitor4.n)
+        connect(capacitor4.n, capacitor5.p)
+        connect(capacitor4.n, resistor2.p)
+        connect(capacitor1.n, capacitor3.n)
+        connect(capacitor1.n, capacitor5.n)
+        connect(resistor2.n, capacitor1.n)
+        connect(resistor1.p, source.p)
+        connect(source.n, ground.g)
+    end
+end
+```
+
+### Defining the Problem
+
+Now the convenience of `@mtkmodel` shines.
+There is no need to declare an `ODESystem` with a `name`, a list of connections, variables and systems.
+The `@mtkmodel` macro has captured all that information for you, and its complement, `@mtkbuild`, will handle the rest.
+
+```julia
+@mtkbuild model = CauerLowPassAnalog()
+```
+
+You now have a `model` which can be used to construct an `ODEProblem`.
+
+```julia
+prob = ODEProblem(model, ModelingToolkit.missing_variable_defaults(model), (0.0, 60.0))
+```
+
+What is happening with `ModelingToolkit.missing_variable_defaults(model)`?
+If you try and run `ODEProblem(model, (0.0, 60.0))` then you will encounter the error:
+
+```plaintext
+ERROR: ArgumentError: Equations (7), states (7), and initial conditions (2) are of different lengths.
+```
+
+This occurs because dummy derivatives where generated without defined initial values.
+Thankfully, `ModelingToolkit` provides `missing_variable_defaults` as a solution to this problem.
+See [ModelingToolkit FAQ](https://docs.sciml.ai/ModelingToolkit/stable/basics/FAQ/) for more information on this and other common questions.
+
+### Solving the Problem
+
+You are finally ready to solve!
+However, if you try to run `solve(prob)` then you will encounter an error.
+
+```plaintext
+ERROR: Default algorithm choices require DifferentialEquations.jl.
+Please specify an algorithm (e.g., `solve(prob, Tsit5())` or
```

Comment:
Perhaps we can suggest `using DifferentialEquations` and `solve(prob)` instead of OrdinaryDiffEq? 
(While keeping the note+url on where they can find all the solvers they can specify if they want to.)

Reply:
You mean for all the tutorials? Seems like a good idea to me. My understanding of the trade-off being `OrdinaryDiffEq.jl` is more lightweight and therefore faster to load, but `DifferentialEquations.jl` is more robust and offers conveniences to new users - which is useful for readers going through the tutorials.

Should I leave the note, and open an issue to switch over all tutorials and update the `Project.toml`?